radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -155,6 +155,7 @@ elseif(PCB STREQUAL X7)
     set(NAVIGATION_TYPE x7)
     set(ROTARY_ENCODER YES)
     set(CPU_TYPE_FULL STM32F407xG)
+    set(PXX2 ON)
```

Comment:
So the T14 and T20V2 will also have PXX2 support?

Reply:
Like the other radios with enough storage.